src/oscar/apps/dashboard/ranges/views.py

```diff
@@ -121,6 +130,13 @@ def get_context_data(self, **kwargs):
         ctx['range'] = range
         if 'form' not in ctx:
             ctx['form'] = self.form_class(range)
```

Comment:
Here, you would set the intiial value of the upload type. Say you named the hidden field: upload_type, you can do this:

```python
ctx['form'] = self.form_class(range, initial={"upload_type": INCLUDED_PRODUCTS_TYPE})
```